plugins/inputs/t128_graphql/t128_graphql.go

```diff
@@ -55,7 +56,24 @@ func (plugin *T128GraphQL) Init() error {
 	if err != nil {
 		return err
 	}
-	plugin.Config = LoadConfig(plugin.EntryPoint, plugin.Fields, plugin.Tags)
+
+	fieldsWithPartialPath, fieldsWithAbsPath, err := validateAndSeparateData(plugin.Fields, plugin.EntryPoint)
```

Comment:
are "partial" paths relative? Could they be named as such?

Reply:
done